package.json

```diff
@@ -15,7 +15,6 @@
     "less": "^3.10.3",
     "less-loader": "^5.0.0",
     "nyc": "^14.1.1",
-    "rc-form": "^2.4.11",
```

Comment:
Why are we removing this? I think it's needed for other components, isn't it?

Reply:
this is strange. i didnt delete this file. i will reinstall it